nipoppy_cli/nipoppy/workflows/pipeline.py

```diff
@@ -250,7 +255,7 @@ def process_template_json(
 
         return template_json_str if return_str else json.loads(template_json_str)
 
-    def get_boutiques_config(self, participant: str, session: str):
+    def get_boutiques_config(self):
```

Comment:
Now that you have got me interested in `cached_properties` I am wondering why is this not one ...